app/src/main/res/anim/activity_open_exit.xml

```diff
@@ -12,7 +12,7 @@
     -->
     <alpha
         android:fromAlpha="1.0"
-        android:toAlpha="1.0"
+        android:toAlpha="0"
```

Comment:
With both toAlpha and fromAlpha 0 this was not doing anything. But maybe that was deliberate by you @charlag?

Reply:
Yeah please see the comment above

Reply:
🤦  changed it again